src/main/java/seedu/realodex/model/person/Birthday.java

```diff
@@ -35,22 +41,50 @@ public Birthday(String birthday) {
         this.birthday = birthdayDate;
     }
 
+    /**
+     * Constructs a default {@code Birthday}.
+     */
     public Birthday() {
-        this.birthday = Optional.empty();
+        SimpleDateFormat formatter = new SimpleDateFormat(INPUT_DATE_PATTERN, Locale.ENGLISH);
+        formatter.setLenient(false);
+        Optional<Date> birthdayDate = Optional.empty();
+        try {
+            birthdayDate = Optional.of(formatter.parse("01May2023"));
```

Comment:
Not sure if putting this as our default birthday will affect sort and filter function?

Reply:
This just exists as placeholder for now, if this is ever called its due to Birthday being a error.